app-text/alanvar/alanvar-0.5.0.recipe

```diff
@@ -0,0 +1,91 @@
+SUMMARY="Al-Anvar, a Quran Study Software "
+DESCRIPTION="Al-Anvar is a cross-platform Quran Study Software. It uses \
+\"Tanzil.net\" and \"Zekr.org\" database as its database."
+
+HOMEPAGE="http://al-anvar.sourceforge.net/"
+COPYRIGHT="2012-2018 S. Hosein Hoseini"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/mazbrili/al-anvar/archive/$portVersion.zip"
+CHECKSUM_SHA256="3f3e57d453302fe11b715e75bce0b2c630cae70143987cf4208c5d8285a52817"
```

Comment:
Suggestion: switch `SOURCE_URI` from `.zip` to `.tar.gz` and add a `SOURCE_FILENAME`:
```diff
 REVISION="1"
-SOURCE_URI="https://github.com/mazbrili/al-anvar/archive/$portVersion.zip"
+SOURCE_URI="https://github.com/mazbrili/al-anvar/archive/$portVersion.tar.gz"
-CHECKSUM_SHA256="3f3e57d453302fe11b715e75bce0b2c630cae70143987cf4208c5d8285a52817"
+CHECKSUM_SHA256="3c7e9c86828416584c54c7a47237078f278056b684ee6ea8d35fec442cb81dca"
+SOURCE_FILENAME="al-anvar-$portVersion.tar.gz"
 SOURCE_DIR="al-anvar-$portVersion"
```
(That's because we prefer to use a tarball instead of a zip archive and because of [HaikuPorter-Guidelines#archive-name](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name).)